newsroom/users/tests.py

```diff
@@ -0,0 +1,30 @@
+from tests import TestCase
+from flask import url_for
+
+
+class UsersTestCases(TestCase):
```

Comment:
would rather put tests into `tests` folder so when this will be installable via pip it won't get them

Reply:
No problem, I'll move them to test folder.